README.md

```diff
@@ -105,10 +104,18 @@ _The above is copied from the [current list of versions](https://www.postgresql.
     - URI's do not work
     - if the `$ref` is broken, the behaviour is considered unexpected
 - Any values which are the `string` `NULL` will be streamed to PostgreSQL as the literal `null`
+- Table identifiers are restricted to:
```

Comment:
Maybe use "Table names ..." here? If you're not used to the nomenclature it may take a second for "identifiers" to click.

Reply:
Good call